bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
   annotations: {{- include "common.tplvalues.render" (dict "value" .Values.annotations "context" $) | nindent 4 }}
   {{- end }}
 spec:
-  serviceName: {{ include "mongodb.fullname" . }}-headless
+  serviceName: {{ include "mongodb.servicename" . }}
```

Comment:
change servicename to servicenameOverride

Reply:
Fixed